javascript/code-challenges/challenges-06.test.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named getNames that, given an array of people objects, uses map to return an array of names reversed.
+
+For example:
+[
+{
+  name: 'lloyd',
+  age: 32,
+  shoeSize: 12
+},
+{
+  name: 'jamie',
+  age: 21,
+  shoeSize: 8
+}
+]
+
+Returns: ['dyoll', 'eimaj'];
+------------------------------------------------------------------------------------------------ */
+
+const getNames = (arr) => {
+  return arr.map(obj => obj.name.split('').
+    reverse('').
+    join(''));
+};
```

Comment:
Nice solution. I appreciate that you put **reverse** and **join** on separate lines for readability sake.